src/index.html

```diff
@@ -84,11 +84,7 @@ <h2 class="f3 fw7 mt3 mb5 purple">A Framework for Change</h2>
     <div class="libre-baskerville lh-2 bg-near-white f5 center bt b--black-10 pv3 ph4 ph5-m ph6-l items-center flex flex-column justify-center">
       <h3 class="p3 open-sans lh-title f2 mt0 mb2 blue tc mt4">Our Sponsors</h3>
       <div class="tc center bg-green w3 pt1 mt3 mb3"></div>
-      <h3 class="p3 libre-baskerville lh-title f3 mt0 mb4 blue tc mt4">Lead Sponsor</h3>
```

Comment:
Would you be able to add this back? I realized my instructions were out of date 🙈 we had this in another section and I had already removed it.